components/library/dbus/patches/10-remove-static-inline-from-headers.patch

```diff
@@ -0,0 +1,55 @@
+--- dbus-1.12.8/dbus/dbus-address.h	2017-10-30 13:26:18.000000000 +0000
```

Comment:
Do I miss something? Have you added two patches to do the same?

Reply:
Sorry about that, botched that patch. Should be fine now.